bindings/c/fdb_c.cpp

```diff
@@ -219,6 +219,11 @@ fdb_error_t fdb_future_get_version( FDBFuture* f, int64_t* out_version ) {
 	CATCH_AND_RETURN( *out_version = TSAV(Version, f)->get(); );
 }
 
+extern "C" DLLEXPORT
+fdb_error_t fdb_future_get_int64( FDBFuture* f, int64_t* out ) {
+	CATCH_AND_RETURN( *out = TSAV(Version, f)->get(); );
```

Comment:
I see `Version` in here. Should this be `int64_t` instead? Also, for consistency with other functions I would suggest renaming the out parameter to something like `out_value`.